src/components/Cart.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import styled from "styled-components";
+import {Modal, Button} from "antd";
+
+const Context = styled.div`
+    display: flex;
+    flex-direction: row;
+    margin: 30px;
+`;
+
+const ItemInfo = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
+const Image = styled.img`
+    height: 150px;    
+`;
+
+const ItemDetail = styled.div`
+    margin: 20px;
+    padding: 10px;
+`;
+
+const GrandTotal = styled.div`
+    display: flex;
+    align-items: flex-end;
+    justify-content: flex-end; 
+`;
+
+export default function Cart({visible, productList, emptyCart, closeCart}){    
+    return(
+        <Modal
+        centered
+        title="Cart"
+        visible={visible}
+        onOk={closeCart}
+        onCancel={closeCart}
+        >
+      {(productList.length == 0)?
+        <p>Your cart is Empty</p>:
```

Comment:
Your formatting looks a bit weird. Did you install Prettier? You could run it automatically on save.
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode